flytekit/__init__.py

```diff
@@ -243,6 +244,7 @@
 from flytekit.core.workflow import ImperativeWorkflow as Workflow
 from flytekit.core.workflow import WorkflowFailurePolicy, reference_workflow, workflow
 from flytekit.deck import Deck
+from flytekit.extras import pydantic
```

Comment:
If we move to lazy import transformer, this will fail to have custom serialize and deserialize behavior, still investigation the root cause.